extensions/wikia/Rail/styles/RailElement.scss

```diff
@@ -0,0 +1,269 @@
+@import "skins/shared/color";
```

Comment:
I'm using githooks ;) and deep selectors are mostly caused by overriding existing properties and selectors.